python/lsst/cp/pipe/utils.py

```diff
@@ -540,6 +540,37 @@ def arrangeFlatsByExpTime(exposureList):
     return flatsAtExpTime
 
 
+def arrangeFlatsByExpId(exposureList):
+    """Arrange exposures by exposure ID.
+
+    Parameters
+    ----------
+    exposureList : `list`[`lsst.afw.image.exposure.exposure.ExposureF`]
+        Input list of exposures.
+
+    Returns
+    ------
+    flatsAtExpTime : `dict` [`float`,
```

Comment:
This seems copied from the other function. Please update, and provide an explanation on the pairs are grouped.  You are sorting by `expId`, and then group the exposures in pairs, correct? For each pair, do the exposures have the same flux? How do we know? Is it guaranteed that this is the case, just by sorting by exposure ID?